UnicodeJsps/src/test/java/org/unicode/jsptest/TestLanguageid.java

```diff
@@ -1,22 +1,22 @@
 package org.unicode.jsptest;
 
+import com.ibm.icu.util.ULocale;
+
+import org.junit.jupiter.api.Test;
 import org.unicode.jsp.LanguageCode;
+import org.unicode.unittest.TestFmwkMinusMinus;
 
-import com.ibm.icu.dev.test.TestFmwk;
-import com.ibm.icu.util.ULocale;
+public class TestLanguageid extends TestFmwkMinusMinus {
 
-public class TestLanguageid extends TestFmwk {
-    public static void main(String[] args) {
-        new TestLanguageid().run(args);
-    }
+    @Test
     public void TestParse() {
         String results;
         results = LanguageCode.validate("pap-CW",new ULocale("en"));
         if (!assertTrue("", results.contains("Curaçao"))) {
             errln(results);
         }
-        
+
         results = LanguageCode.validate("$, eng-840, fr-fr",new ULocale("en"));
-        assertTrue("", results.contains("target='languageid'>fr-FR</b>"));
+        assertContains(results, "target='languageid'>fr-FR</a></b>");
```

Comment:
@macchiati  this seemed like a simple data fix in the assertion, maybe the HTML just changed?